content/articles/account-invoice-history.md

```diff
@@ -36,15 +36,15 @@ Your invoices may be in certain states depending on different circumstances:
 
 ### `open` {#invoice-state-open}
 
-An invoice in an `open` state is being prepared by our systems and is not ready for collection yet. It's unlikely that you'll see open invoices, as they only remain in this state for short periods of time.
+An invoice in an `open` state is being prepared and isn't ready for collection yet. It's unlikely that you'll see open invoices, as they only remain in this state for short periods of time.
 
 ### `collected` {#invoice-state-collected}
 
-When an invoice has been paid for, it'll be in the `collected` state.
+An invoice in the `collected` state indicates it has been paid.
 
 ### `failed` {#invoice-state-failed}
 
-When you make a purchase, like a domain registration or an SSL certificate, and we're unable to collect the payment, we'll notify you via email. The invoice will be marked as `failed`.
+When you make a [purchase](/articles/understanding-invoice/#purchases), like a domain registration or an SSL certificate, and we're unable to collect the payment, we'll notify the account owner via email. The invoice will be marked as `failed`.
 
 While the invoice is in this state, you won't be able to make other purchases. You can [retry the payment](#retrying) yourself, and our systems will also try to collect it every few hours.
```

Comment:
"Every few hours" but do we know for how long?

Reply:
I do not have a specific number. It seems variable. Maybe we should put something like " and our systems will also try to collect it automatically again."